change/@microsoft-teams-js-24933635-de42-4b04-b53f-935c1799929a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "draft solution with optional apiVersion",
+  "packageName": "@microsoft/teams-js",
+  "email": "email not defined",
```

Comment:
this should be the email address associated with your github account.

Reply:
Added. But it seems like in Mac I need to manually add this. It cannot relate to my email automatically, which is odd.